compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpServiceValidator.java

```diff
@@ -79,6 +93,48 @@ public void perform(SyntaxNodeAnalysisContext syntaxNodeAnalysisContext) {
         }
     }
 
+    private static void extractServiceAnnotationAndValidate(SyntaxNodeAnalysisContext ctx,
+                                                            ServiceDeclarationNode serviceDeclarationNode) {
+        Optional<MetadataNode> metadataNodeOptional = serviceDeclarationNode.metadata();
+
+        if (metadataNodeOptional.isEmpty()) {
+            return;
+        } else {
```

Comment:
```suggestion
        }
```
Can shift the code 1 tab to the left